scripts/upgrade

```diff
@@ -253,6 +260,17 @@ pushd "$final_path"
 	ynh_exec_warn_less sudo -u $app env $ynh_node_load_PATH NODE_CONFIG_DIR="$final_path/config" NODE_ENV=production $ynh_npm run plugin:install -- --npm-name peertube-plugin-auth-ldap
 popd
 
+#=================================================
+# PEERTUBE 4.0.0 UPGRADE MIGRATION SCRIPT
+#=================================================
+
+if ynh_compare_current_package_version --comparison lt --version 4.0.0~ynh1; then
+	ynh_script_progression --message="Running Peertube 4.0.0 migration script..."
+	pushd "$final_path"
+		ynh_exec_warn_less sudo -u $app env $ynh_node_load_PATH NODE_CONFIG_DIR="$final_path/config" NODE_ENV=production node dist/scripts/migrations/peertube-4.0.js
+	popd
+fi
```

Comment:
I need to do some check of this… The migration script *may* need the service to be running.